gitutils/__init__.py

```diff
@@ -21,10 +21,10 @@ def execute_command(command, output=True, prefix='', grep=None):
 
     p = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, bufsize=-1)
     (cmdout, cmderr) = p.communicate()
-    if cmdout is None:
-        return (0, "")
+    if cmdout is None or not cmdout.strip():
+        return (p.returncode, cmdout.strip())
```

Comment:
More testing revealed a bug in this line. The return should be like this:

```python
+        return (not p.returncode, cmdout.strip())
```

The results of the bug is output like this:

```
$ git multi -g
--------------------------------------------------------------------------------
Executing git status -s
--------------------------------------------------------------------------------
Progress: ## Using project list from directory search: /home/troth/git/git-plus/foo
* * * *
--------------------------------------------------------------------------------

test-1, test-2, test-3, test-4:
	Error executing:['git', 'status', '-s']
```

When the expected output is this:

```
$ git multi -g
--------------------------------------------------------------------------------
Executing git status -s
--------------------------------------------------------------------------------
Progress: ## Using project list from directory search: /home/troth/git/git-plus/foo
* * * *
--------------------------------------------------------------------------------

test-1, test-2, test-3, test-4:
	OK
```

I will integrate a fix for this and submit a reworked patchset.